src/db-service/shopping_cart.js

```diff
@@ -10,7 +10,7 @@ exports.createCart = async (name, email) => {
   return result[0];
 };
```

Comment:
Oracle database has a dead connection detection mechanism but we should call connection.close before set the connection.__knex__disposed.